Registry/Registry.EM_COMMON

```diff
@@ -2910,7 +2915,7 @@ package   mynnpblscheme3_dfi bl_pbl_physics_dfi==6   -             dfi_scalar:df
 
 package   nocuscheme     cu_physics==0               -             -
 package   kfetascheme    cu_physics==1               -             state:w0avg
-package   bmjscheme      cu_physics==2               -             -
+package   bmjscheme      cu_physics==2               -             state:qcconv,qiconv
```

Comment:
@rmfonsecaweb Should add CONVCLD, CCLDFRA to the package too.

Reply:
Done!